pkg/console/subresource/deployment/deployment.go

```diff
@@ -189,6 +196,14 @@ func withConsoleAnnotations( // FIXME: should probably add authnConfig annotatio
 		deployment.ObjectMeta.Annotations[oauthServingCertConfigMapResourceVersionAnnotation] = oauthServingCertConfigMap.GetResourceVersion()
 	}
 
+	if authConfig != nil {
```

Comment:
This should never be nil.

Do we really need to rollout the deployment on any change of this config? Wouldn't the changes be projected into the configuration ConfigMap and client secret's Secret?

Reply:
You're right -- I'll remove this annotation.